web/js/main.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```

Comment:
Maybe we should add a comment on this line that explains why we need to import it;
```
import 'babel-polyfill'; // Needed for worldview-components in IE and older browsers
```